File: Python/ConstructTheRectangle/ConstructTheRectangle.py
import math
class Solution(object):
	# (39ms)
	def constructRectangle(self, area):
		"""
		:type area: int
		:rtype: List[int]
		"""
		width = int(math.sqrt(area))
		while area % width != 0:
			width -= 1
		return [area / width, width]

print Solution().constructRectangle(4)

File: Java/ConstructTheRectangle/ConstructTheRectangle.java
/**
 * (6ms)
 * 很简单的一道题。
 * @author Phoenix-Z
 *
 */
public class ConstructTheRectangle {

	public int[] constructRectangle(int area) {
		int width = (int)Math.sqrt(area), length = area / width;
		while(width * length != area) {
			width--;
			length = area / width;
		}
		return new int[] {length, width};
    }
	
}
